src/components/study-pane.js

```diff
@@ -354,87 +355,44 @@ const StudyPane = (props) => {
 
     const [position, setPosition] = useState([-1, -1]);
 
-    const loadNetwork = useCallback(() => {
-        console.info(`Loading network of study '${studyName}'...`);
-        updateLoadFlowResult();
-        updateSecurityAnalysisResult();
-        updateSecurityAnalysisStatus();
-
-        const substations = fetchSubstations(studyName, userId);
-        const lines = fetchLines(studyName, userId);
-        const twoWindingsTransformers = fetchTwoWindingsTransformers(
-            studyName,
-            userId
-        );
-        const threeWindingsTransformers = fetchThreeWindingsTransformers(
-            studyName,
-            userId
-        );
-        const generators = fetchGenerators(studyName, userId);
-        const loads = fetchLoads(studyName, userId);
-        const batteries = fetchBatteries(studyName, userId);
-        const danglingLines = fetchDanglingLines(studyName, userId);
-        const hvdcLines = fetchHvdcLines(studyName, userId);
-        const lccConverterStations = fetchLccConverterStations(
-            studyName,
-            userId
-        );
-        const vscConverterStations = fetchVscConverterStations(
-            studyName,
-            userId
-        );
-        const shuntCompensators = fetchShuntCompensators(studyName, userId);
-        const staticVarCompensators = fetchStaticVarCompensators(
-            studyName,
-            userId
-        );
-
-        Promise.all([
-            substations,
-            lines,
-            twoWindingsTransformers,
-            threeWindingsTransformers,
-            generators,
-            loads,
-            batteries,
-            danglingLines,
-            hvdcLines,
-            lccConverterStations,
-            vscConverterStations,
-            shuntCompensators,
-            staticVarCompensators,
-        ])
-            .then((values) => {
-                const network = new Network();
-                network.setSubstations(values[0]);
-                network.setLines(values[1]);
-                network.setTwoWindingsTransformers(values[2]);
-                network.setThreeWindingsTransformers(values[3]);
-                network.setGenerators(values[4]);
-                network.setLoads(values[5]);
-                network.setBatteries(values[6]);
-                network.setDanglingLines(values[7]);
-                network.setHvdcLines(values[8]);
-                network.setLccConverterStations(values[9]);
-                network.setVscConverterStations(values[10]);
-                network.setShuntCompensators(values[11]);
-                network.setStaticVarCompensators(values[12]);
-
+    const loadNetwork = useCallback(
+        (isJusteUpdate) => {
```

Comment:
isJustUpdate or isUpdate for argument name

Reply:
done